.github/workflows/build.yml

```
@@ -7,21 +7,131 @@ on:
+              linux/arm64
+            tags: |-
+              jrei/systemd-debian:11
+              jrei/systemd-debian:bullseye
+          - name: systemd-debian-sid
+            dockerfile: systemd/debian/sid.Dockerfile
+            platforms: |-
+              linux/amd64
+              linux/arm/v7
+              linux/arm64
+            tags: |-
+              jrei/systemd-debian:sid
+          - name: systemd-fedora-latest
+            dockerfile: systemd/fedora/Dockerfile
+            platforms: |-
+              linux/amd64
+              linux/arm/v7
+              linux/arm64
+            tags: |-
+              jrei/systemd-fedora:latest
+          - name: systemd-ubuntu-16.04
+            dockerfile: systemd/ubuntu/16.04.Dockerfile
+            platforms: |-
+              linux/amd64
+              linux/arm/v7
+              linux/arm64
+            tags: |-
+              jrei/systemd-ubuntu:16.04
+              jrei/systemd-ubuntu:xenial
+          - name: systemd-ubuntu-18.04
+            dockerfile: systemd/ubuntu/18.04.Dockerfile
+            platforms: |-
+              linux/amd64
+              linux/arm/v7
+              linux/arm64
+            tags: |-
+              jrei/systemd-ubuntu:18.04
+              jrei/systemd-ubuntu:bionic
+          - name: systemd-ubuntu-20.04
+            dockerfile: systemd/ubuntu/20.04.Dockerfile
+            platforms: |-
+              linux/amd64
+              linux/arm/v7
+              linux/arm64
+            tags: |-
+              jrei/systemd-ubuntu:20.04
+              jrei/systemd-ubuntu:focal
+              jrei/systemd-ubuntu:latest
     steps:
-      - name: Trigger builds
-        run: |
-          set -e
-          # systemd-debian
-          curl -sXPOST https://hub.docker.com/api/build/v1/source/7ff4cb81-1669-4d8c-b71c-78d372008ca6/trigger/a713677f-000c-4d36-b10e-43ddaaba1f44/call/
-          # systemd-ubuntu
-          curl -sXPOST https://hub.docker.com/api/build/v1/source/e46dc792-7434-460a-a604-c2f550a4c1d9/trigger/eae24f56-bcdc-473f-a75c-05ac4b9cd399/call/
-          # systemd-centos
-          curl -sXPOST https://hub.docker.com/api/build/v1/source/2a412a1d-c4d0-4818-bf65-f695280494e3/trigger/d724213d-1aec-458e-9000-3075263bf6ec/call/
-          # systemd-fedora
-          curl -sXPOST https://hub.docker.com/api/build/v1/source/3d0619f4-95ad-4629-9995-4534e06a0076/trigger/30747760-509f-4e89-a0be-5941469a922e/call/
-          # openrc-alpine
-          curl -sXPOST https://hub.docker.com/api/build/v1/source/dafbfb54-1ba0-43f7-8ec5-de1fe9d1ed1a/trigger/4a4c2852-5093-47e3-b754-44eb8e9a46c5/call/
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1 
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
```

```suggestion
          username: jrei
```
I've seen this in the official example, but no need: not a secret.

---

Ok, fixed :)